client-app/pages/product.vue

```diff
@@ -67,6 +73,7 @@ import { CarouselProductCard, useProduct, useRelatedProducts, useCategory } from
 import { BackButtonInHeader } from "@/shared/layout";
 import { useTemplate } from "@/shared/static-content";
 import type { Breadcrumb } from "@/core/api/graphql/types";
+import VcCopyText from "@/ui-kit/components/molecules/copy-text/vc-copy-text.vue";
```

Comment:
You don't need this because all components are registered globally. But to make it visible in IDE we need something like this: https://github.com/VirtoCommerce/vc-shell/blob/main/framework/ui/types/index.ts
I think it's a good point for yet another tech dept